samcli/commands/local/start_api/cli.py

```diff
@@ -32,6 +32,13 @@
  compiled languages or projects requiring complex packing support, we recommended you run your own building solution
 and point SAM to the directory or file containing build artifacts.
 """
+COMMAND_SUGGESTIONS = """
+Commands you can use next
+=========================
+[*]Validate SAM template: sam validate
```

Comment:
Missing space (also the other help text).

Is this supposed to be verbatim the message that shows up on `sam init`? If yes, should this be a shared variable instead? So next time we add some command/parameter we want customers to use we don't need to remember updating in multiple places.

Reply:
its similiar to what sam init but the words are different.

Reply:
How are these different from the one in `sam init`?

(Comment about missing spaces after `[*]` still applies)

Reply:
for sam init there is another line: [*] Create pipeline: cd hello-world-for-command-suggestions && sam pipeline init --bootstrap rather than [*]Deploy: sam deploy --guided